dockstore-webservice/scripts/sortOpenAPIYaml.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+yq r --tojson src/main/resources/openapi3/unsortedopenapi.yaml | yq r - -P > src/main/resources/openapi3/openapi.yaml
```

Comment:
Code comment, what does `-P` do?
Ideally, link to doc (might have missed it, but didn't find it in https://kislyuk.github.io/yq/ )

Reply:
https://github.com/mikefarah/yq